dlgr/griduniverse/game_config.yml

```diff
@@ -328,3 +348,16 @@ transitions:
       - -1
     target_end: empty_gooseberry_bush
     target_start: gooseberry_bush
+
+  # At least two players can use a sharp stone to consume a stag
+  - actor_start: sharp_stone
+    actor_end: sharp_stone
+    required_actors: 2
+    target_start: stag
+    target_end: fallen_stag
+    visible: always
+    last_use: false
+    calories: 20
```

Comment:
The idea here is that consuming the stag provides a total of 20 calories, split evenly among the participants (with the initiator getting any remainder), and what's left is a fallen stag which may have separate uses?

Reply:
Maybe we should have another example multi-actor transition with no calories, but results in an object with multiple available uses?

Reply:
@alecpm You're correct about what the idea is.

Items added to this file are effective in the game. I can add one more example, but it should probably be a "real" one. Or maybe I misunderstood your suggestion.

Reply:
I agree it would be good to have a "real" example of such a transition, but such transitions are the second checkbox on #235. So I think we'd ideally have such an example available. Maybe best to ask Natalia for one?